huxley/api/views/user.py

```diff
@@ -22,6 +23,11 @@ class UserList(generics.ListCreateAPIView):
     queryset = User.objects.all()
     permission_classes = (IsPostOrSuperuserOnly,)
 
+    def create(self, request, *args, **kwargs):
+        if settings.REGISTRATION_OPEN:
+            return super(UserList, self).create(request, args, kwargs)
```

Comment:
You have to explode the args and kwargs when you call the super method, otherwise they're passed in as just regular params as a tuple and dictionary, respectively.
```python
return super(UserList, self).create(request, *args, **kwargs)
```